### ISSUE TITLE
Getting started tutorial needs updating

### ISSUE THREAD

**Issue by JamieJQuinn** [spcl/dace#1011]
**Compiled SDFG does not require keyword args**

In the [Getting Started tutorial](https://nbviewer.org/github/spcl/dace/blob/master/tutorials/getting_started.ipynb), just before code cell 11, the notebook states:

"A compiled SDFG, however, has to be invoked like an SDFG, with keyword arguments only:"

This does not seem to be correct; I can run the line fine with `A` as a positional argument.

**Final SDFG does not match latest dace output**

In the [Explicit Dataflow example](https://nbviewer.org/github/spcl/dace/blob/master/tutorials/getting_started.ipynb#Explicit-Dataflow) the outputted graph does not match that produced using Dace 0.13.1 and Python 3.10.4:

[sse_sigma.txt](https://github.com/spcl/dace/files/8805690/sse_sigma.txt)

**Comment by tbennun** [comment]
You are correct, if the SDFG was generated from one of the frontends (Python for example) it retains the order of the arguments. However, if the SDFG API is used, and `sdfg.arg_names` is not provided, keyword arguments must still be used. We'll clarify, thanks for reporting!
